core/oslc4j-jena-provider/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/AbstractOslcRdfXmlProvider.java

```diff
@@ -146,6 +147,7 @@ private void writeObjectsTo(final Object[] objects, final OutputStream outputStr
 			writer.write(model,
 						 outputStream,
 						 null);
+            log.trace("writeObjectsTo");
```

Comment:
How about using nanoTime twice as described in https://www.baeldung.com/java-measure-elapsed-time

Reply:
Actually, I think Java 8+ `Instant` is even better.

Reply:
done.